### PR TITLE
Docs - Suspend

### PR DESCRIPTION
**Pull request by RakshithaK06**
Story details: https://app.clubhouse.io/kintohub/story/721

Added a content for "Suspend Service" feature in "Settings" section.

### CODE REVIEW

File: docs/anatomy/anatomy-settings.md [hunk 1]
```diff
@@ -22,13 +22,34 @@ Public APIs are available only for [Backend Services](../service-types/types-bac
...
+![Suspend Service](/img/anatomy/suspend-service.png)
+
+2. Click on the **Confirm** button in the prompt displayed.
+
+![Confirm suspend](/img/anatomy/confirm-suspend-service.png)
+
+Once you confirm, the services will stop running and status will change to `Suspended`.
+
+:::note
+To revoke the suspension, all you need to do is click on the __Resume__ button. Once you resume, the suspended services will be redeployed on KintoHub.
```
Comment (Disturbing) [inline docs/anatomy/anatomy-settings.md:48]: Action - Add docs to the Releases page and link that you can also resume it from the releases page.

Comment (Disturbing) [reply docs/anatomy/anatomy-settings.md:48]: To resume your service, click on the ....

Change the above text to that, keeps it more simple, less conversational.

**Review by Disturbing — CHANGES_REQUESTED** [review]
Missing docs for Releases tab

File: docs/anatomy/anatomy-settings.md (hunk 1 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-settings.md:48]: Done

**Review by Disturbing — APPROVED** [review]

**Pull request merged**